analyzer/block/block.go

```diff
@@ -22,8 +22,8 @@ import (
 const (
 	// Timeout to process a block.
 	processBlockTimeout = 61 * time.Second
-	// Number of blocks to be processed in a batch.
-	blocksBatchSize = 100
+	// Default number of blocks to be processed in a batch.
+	defaultBatchSize = 100
```

Comment:
Should we change this to 1000 or even 5000? Now that we have the cache in place (and it's prewarmed and we can copy it around with some ops busywork), it's more likely that we'll read from the cache than from the node. It's also fine if the batch is too large (for the batch timeout); it just means we'll lock some blocks too many, but the lock will auto-release soon enough.

Reply:
Updated to 1000.